tests/cover/test_renaming.py

```diff
@@ -0,0 +1,28 @@
+# coding=utf-8
+#
+# This file is part of Hypothesis, which may be found at
+# https://github.com/HypothesisWorks/hypothesis-python
+#
+# Most of this work is copyright (C) 2013-2017 David R. MacIver
+# (david@drmaciver.com), but it contains contributions by others. See
+# CONTRIBUTING.rst for a full list of people who may hold copyright, and
+# consult the git log if you need to determine who owns an individual
+# contribution.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v. 2.0. If a copy of the MPL was not distributed with this file, You can
+# obtain one at http://mozilla.org/MPL/2.0/.
+#
+# END HEADER
+
+from __future__ import division, print_function, absolute_import
+
+from hypothesis.internal.renaming import renamed_arguments
+
+
+def test_can_rename_arguments_in_a_function_with_no_docstring():
+    @renamed_arguments(old_arg='new_arg')
```

Comment:
So the usage here is different from its usage in the actual API, where both the old and the new argument are present in the underlying docstring. I guess this usage works too, but it's not what I was expecting.

Also would this test be better off returning the value and then making an assertion about it rather than printing?

Reply:
I don’t think I understand your first comment.

I’ll make the test more interesting, per your second comment.

Reply:
In our actual usage we have:

```python
@renamed_argument(b='a')
def f(a, b):
    pass
```

In your usage we have:


```python
@renamed_argument(b='a')
def f(a):
    pass
```

i.e. the argspec does not contain the original name of the argument in this test.

(The reason for the usage pattern with both of them in is that it makes backwards-compatibility of argument positions more obvious)

Reply:
Ah, gotcha.